ios/CONTRIBUTING.md

```diff
@@ -8,11 +8,11 @@ To build and run the demo app follow these steps:
 - Clone the [FluentUI](https://github.com/microsoft/fluentui-apple) repository.
 - Open `ios/FluentUI.xcworkspace` in Xcode.
 - In the Xcode scheme menu choose `Demo.development` and choose an iOS Simulator(or Device if you have your own device provisioning profile) to deploy to.
-- Make sure Swift Package Manager has download 2 dependencies(appcenter-sdk-apple and PLCrashReporter) for demo app.
+- Make sure Swift Package Manager has downloaded 2 dependencies(AppCenter and PLCrashReporter) for demo app.
```

Comment:
Maybe this should stay as `appcenter-sdk-apple`. Isn't that the actual name of the package? Also there should be a space between `despendencies` and `(`.

Reply:
I see only `AppCenter` as the name of the package and that is how the dependencies are list in Xcode navigation pane as well. 
![CleanShot 2024-01-26 at 07 06 57@2x](https://github.com/microsoft/fluentui-apple/assets/46041492/49fc485e-078b-45e2-b82c-a1b39b67fd0c)

Reply:
IMO, space is optional there.